demo/src/test/java/com/mongodb/jdbc/utils/TestUtils.java

```diff
@@ -0,0 +1,439 @@
+        ResultSet rs = m_dbMeta.getColumnPrivileges(null,null,"myTable","%");
+        printRsInfo(rs);
+    }
+
+    @Test
+    void testGetCrossReference() throws Exception
+    {
+        //ResultSet rs = m_dbMeta.getCrossReference(null,null,null,null,null, null);
+        ResultSet rs = m_dbMeta.getCrossReference(null,null,"parentTable","foreignCatalog",null,"foreignTable");
+        printRsInfo(rs);
+    }
+
+    @Test
+    void testGetExportedKeys() throws Exception
+    {
+        //ResultSet rs = m_dbMeta.getExportedKeys(null,null,null);
+        ResultSet rs = m_dbMeta.getExportedKeys(null,null,"myTable");
+        printRsInfo(rs);
+    }
+
+    @Test
+    void getImportedKeys() throws Exception
+    {
+        //ResultSet rs = m_dbMeta.getImportedKeys(null,null,null);
+        ResultSet rs = m_dbMeta.getImportedKeys(null,null,"myTable");
+        printRsInfo(rs);
+    }
+
+    @Test
+    void testGetIndexInfo() throws Exception
+    {
+        ResultSet rs = m_dbMeta.getIndexInfo(null,null,null,false, false);
+        printRsInfo(rs);
+    }
+
+    @Test
+    void testGetPrimaryKeys() throws Exception
+    {
+        //ResultSet rs = m_dbMeta.getPrimaryKeys(null,null,null);
+        ResultSet rs = m_dbMeta.getPrimaryKeys(null,null,"myTable");
+        printRsInfo(rs);
+    }
+
+    @Test
+    void testGetPseudoColumns() throws Exception
+    {
+        //ResultSet rs = m_dbMeta.getPseudoColumns(null,null,null,null);
+        ResultSet rs = m_dbMeta.getPseudoColumns(null,"%","%","%");
+        printRsInfo(rs);
+    }
+
+    @Test
+    void testGetSuperTables() throws Exception
+    {
+        ResultSet rs = m_dbMeta.getSuperTables(null,null,null);
+        printRsInfo(rs);
+    }
+
+    @Test
+    void testGetSuperTypes() throws Exception
+    {
+        ResultSet rs = m_dbMeta.getSuperTypes(null,null,null);
+        printRsInfo(rs);
+    }
+
+    @Test
+    void testGetTypeInfo() throws Exception
+    {
+        ResultSet rs = m_dbMeta.getTypeInfo();
+        printRsInfo(rs);
+    }
+
+    @Test
+    void testGetUDTs() throws Exception
+    {
+        ResultSet rs = m_dbMeta.getUDTs(null,null,null, new int[]{1});
+        printRsInfo(rs);
+    }
+
+    @Test
+    void testDBMeta() throws Exception {
+        System.out
+                .println("-----------------------------------------------------------------------");
+        System.out
+                .println("-----------------------------------------------------------------------");
+        System.out.println("API databases matadata test call:");
+        System.out.println("Driver Name: " + m_dbMeta.getDriverName());
+        System.out.println("Driver Version: " + m_dbMeta.getDriverVersion());
+        System.out.println("Driver Major Version: "
+                + m_dbMeta.getDriverMajorVersion());
+        System.out.println("Driver Minor Version: "
+                + m_dbMeta.getDriverMinorVersion());
+        System.out.println("Server product Name: "
```

Comment:
[nit] Can we make the capitalization more consistent across all of the strings in this code section?